photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataSet.scala

```diff
@@ -73,15 +78,29 @@ protected[ml] class RandomEffectDataSet(
     update(updatedActiveData, updatedPassiveDataOption)
   }
 
+  /**
+   *
+   * @return The Spark context
+   */
   override def sparkContext: SparkContext = activeData.sparkContext
```

Comment:
Perfect - again I am not asking to fix everything, but active/passive is important, and it's a very contained improvement, so let's do it now and be done with it.